Books/server/src/config/index.ts

```diff
@@ -13,7 +13,8 @@ const {
   POSTGRES_PASSWORD,
   BCRYPT_PASSWORD,
   SALT_TIMES,
-  TOKEN_SECRET
+  TOKEN_SECRET,
+  SECOND_TOKEN_SECRET
```

Comment:
I'm not quite clear on what the use of this variable is, do you mind explaining it to me?